build.sbt

```diff
@@ -86,7 +100,16 @@ def playProject(projectName: String, devHttpPorts: Map[String, String]) =
         s"-J-Dlogs.home=/var/log/${packageName.value}",
         s"-J-Xloggc:/var/log/${packageName.value}/gc.log"
       ),
-      commonSettings
+      commonSettings,
+      initialize := {
```

Comment:
Is there a reason we take this step in `sbt`, rather than the shell script? Maybe it's a pain to script?

Reply:
We perform this step in both script _and_ sbt. This might be overkill, but I'm thinking there are a few cases when we run the app / tests / compilation etc. in sbt. In those cases the script checks won't save us.

You could argue this means we should get rid of the script checks, but I feel those are useful to catch the java version error early.